db/migrate/20240312201146_migrate_task_descriptions_fix_kramdown_quotes.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class MigrateTaskDescriptionsFixKramdownQuotes < ActiveRecord::Migration[7.1]
+  def change
+    Task.find_each {|task| task.update(description: fix_kramdown_descriptions(task.description)) }
```

Comment:
Just raising the question here again: Do we want to be notified about validation errors by using `update!` rather than `update`?